nemo/collections/nlp/modules/common/transformer/transformer_decoders.py

```diff
@@ -63,6 +66,7 @@ def __init__(
         self.layer_norm_3 = nn.LayerNorm(hidden_size, eps=1e-5)
         self.third_sub_layer = PositionWiseFF(hidden_size, inner_size, ffn_dropout, hidden_act)
 
+    # TODO: add Neural Types
```

Comment:
Add before merge

Reply:
I have another PR I'm working on where I'm adding this. Can we just add it there?

Reply:
Sounds good